index.js

```diff
@@ -0,0 +1,192 @@
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const inquirer = require("inquirer");
+const fs = require("fs");
+const generateTeam = require("./src/generateTeam");
+const { ensureExpectedIsNonNegativeInteger } = require("jest-matcher-utils");
+
+team = [];
+
+const managerQuestions = () => {
+  inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "name",
+        message: "What is the team manager's name?",
+      },
+      {
+        type: "input",
+        name: "id",
+        message: "What is the team manager's ID?",
+      },
+      {
+        type: "input",
+        name: "email",
+        message: "What is the team manager's e-mail address?",
+      },
+      {
+        type: "input",
+        name: "officeNumber",
+        message: "What is the team manager's office number?",
+      },
+      {
+        type: "list",
+        name: "addTeamMember",
+        message: "Which team member/s would you like to add?",
+        choices: [
+          "Engineer",
+          "Intern",
+          "I don't want to add any more members.",
+        ],
+      },
+    ])
+    .then((managerAnswers) => {
```

Comment:
Try to start moving away from the .then() method and replace it with async/await. Let us know if you'd like some help with it/ we should organise a workshop to explain it in more depth :-)